src/test/scala/thurloe/service/ThurloeServiceSpec.scala

```diff
@@ -27,7 +27,7 @@ class ThurloeServiceSpec extends FunSpec with ScalatestRouteTest {
 
   describe("The Thurloe Service") {
     it("should allow key/value pairs to be set") {
-      Post(uriPrefix, ukvp1) ~> thurloeService.routes ~> check {
+      Post(uriPrefix, ukvp1) ~> thurloeService.keyValuePairRoutes ~> check {
```

Comment:
I had these being scanned by the entire routing sequence to check the routing was working holistically.